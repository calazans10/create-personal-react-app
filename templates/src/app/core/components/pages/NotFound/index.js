import React from 'react';
import MainContent from '../../modules/MainContent';

const NotFound = () => (
  <MainContent title="Página não encontrada">
    <p>Não foi possível encontrar uma página com o caminho informado.</p>
  </MainContent>
);

export default NotFound;
