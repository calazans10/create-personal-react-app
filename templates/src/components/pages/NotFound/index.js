import React from 'react';

const NotFound = () => (
  <div>
    <h1>Page not found</h1>
    <p>We are sorry but the page you are looking for does not exist.</p>
  </div>
);

export default NotFound;
