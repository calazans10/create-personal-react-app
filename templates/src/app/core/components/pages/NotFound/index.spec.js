import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from './index';

describe('NotFound', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<NotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
