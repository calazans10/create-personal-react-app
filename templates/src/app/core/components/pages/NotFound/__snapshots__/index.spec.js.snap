// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders without crashing 1`] = `
<MainContent
  title="Página não encontrada"
>
  <p>
    Não foi possível encontrar uma página com o caminho informado.
  </p>
</MainContent>
`;
